content/en/tracing/trace_collection/library_config/dotnet-framework.md

```diff
@@ -240,19 +220,70 @@ The `from-key` value is specific to the integration type, and should exclude the
+**Default**: `31`
+
+`DD_TRACE_LOGGING_RATE`
+: Sets rate limiting for log messages. If set, unique log lines are written once per `x` seconds. For example, to log a given message once per 60 seconds, set to `60`. Setting to `0` disables log rate limiting. Added in version 1.24.0. Disabled by default.
+
+#### OpenTelemetry
+
 `DD_TRACE_OTEL_ENABLED`
 : Enables or disables OpenTelemetry based tracing, both for [custom][18] or [automatic][19] instrumentation.
 Valid values are: `true` or `false`.<br>
 **Default**: `false`
 
-#### Automatic instrumentation optional configuration
+### Automatic instrumentation optional configuration
 
 The following configuration variables are available **only** when using automatic instrumentation:
 
+#### Traces
+
+`DD_TRACE_EXPAND_ROUTE_TEMPLATES_ENABLED`
+: Expands all route parameters in the application for ASP.NET/ASP.NET Core (except ID parameters)<br>
+This can be useful if you are using parameter names to differentiate between form values, or a slug, such as in GraphQL.
+**Default**: `false`
+Added in version 2.5.2
+
+`DD_TRACE_METHODS`
+: List of methods to trace. Accepts a semicolon (`;`) separated list where each entry has the format `TypeName[MethodNames]`, where `MethodNames` is either a comma (`,`) separated list of method names or the `*` wildcard. For generic types, replace the angled brackets and the type parameters' names with a backtick (`` ` ``) followed by the number of generic type parameters. For example, `Dictionary<TKey, TValue>` must be written as `` Dictionary`2 ``. For generic methods, you only need to specify the method name. <br>
+**Example**: ```Namespace1.Class1[Method1,GenericMethod];Namespace1.GenericTypeWithOneTypeVariable`1[ExecuteAsync];Namespace2.Class2[*]```<br>
+**Note:** The wildcard method support (`[*]`) selects all methods in a type except constructors, property getters and setters, `Equals`, `Finalize`, `GetHashCode`, and `ToString`. <br>
+Added in version 2.6.0.
+Wildcard support `[*]` added in version 2.7.0.
+
+`DD_TRACE_KAFKA_CREATE_CONSUMER_SCOPE_ENABLED`
+: Alters the behavior of the Kafka consumer span<br>
+**Default**: `true`<br>
+When set to `true`, the consumer span is created when a message is consumed and closed before consuming the next message. The span duration is representative of the computation between one message consumption and the next. Use this setting when message consumption is performed in a loop. <br>
+When set to `false`, the consumer span is created when a message is consumed and immediately closed. Use this setting when a message is not processed completely before consuming the next one, or when multiple messages are consumed at once. When you set this parameter to `false`, consumer spans are closed right away. If you have child spans to trace, you must extract the context manually. Read [Headers extraction and injection][12] for more details.
+
+#### Agent
```

Comment:
Agent doesn't seem like the correct section header to me. At a high-level, the current configurations that are listed here are:
- `DD_TRACE_ENABLED` which turns on and off tracing. Perhaps better suited for the Traces section?
- `DD_DBM_PROPAGATION_MODE` which configures the integration between APM and DBM. Maybe this could be in its own DBM section?
- `DD_RUNTIME_METRICS_ENABLED` which configures runtime metrics, which is its own feature. Maybe this could be in its own Runtime Metrics section?

Reply:
Good points! This actually aligns better with the other languages.